python/ray/function_manager.py

```diff
@@ -121,22 +120,20 @@ def from_function(cls, function):
         function_name = function.__name__
         class_name = ""
 
-        function_source_hasher = hashlib.sha1()
+        pickled_function_hasher = hashlib.sha1()
         try:
```

Comment:
this code structure is no longer relevant, please just make it a single line that pickles and computes the hash

Reply:
don't need the `except`